src/popup.html

```diff
@@ -18,7 +18,7 @@
         </svg>
       </div>
       <div class="file-input">
-        <input type="file" id="sticker-file" accept="image/png" class="actual-file-input">
```

Comment:
make the mimetype `image/*` instead of removing it all together, so that people do not try uploading files other than images